pkgs/development/python-modules/mkdocs-drawio-exporter/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchPypi
+, isPy3k
+, lib
+, mkdocs
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-drawio-exporter";
+  version = "0.8.0";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-b079XmbZIZvIiCInwv4S+lqOnLpzZVKZP69eYQWVahA=";
+  };
+
+  propagatedBuildInputs = [ mkdocs ];
```

Comment:
Reading the docs and source of this plugin i have seen that it's possible to define the drawio executable, but if you don't provide one they will search in the PATH environment: https://github.com/LukeCarrier/mkdocs-drawio-exporter/blob/master/mkdocsdrawioexporter/exporter.py#L192

In this case we can simply add `drawio-headless` to the propagated build inputs. This provides the user a working default config so they don't *have to* (but still *can* override via config) define a drawio executable path

Reply:
Makes sense - adopted your suggestion with commit [6c831aa](https://github.com/NixOS/nixpkgs/pull/173730/commits/6c831aa8471072dd95a40bf7f74970f4609fd25f).

Reply:
Ok thank you. please squash that commit into your init-pkg commit and then it looks like good to go.